frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.tsx

```diff
@@ -180,8 +180,6 @@ export const PublicOrEmbeddedQuestion = ({
       parameterValues={parameterValues}
       setParameterValue={setParameterValue}
       setParameterValueToDefault={setParameterValueToDefault}
-      // We don't support background: false on questions (metabase#43838)
-      background
```

Comment:
logic moved to `frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.tsx`